test/RewriteSearch/Polynomial.lean

```diff
@@ -174,7 +174,7 @@ example {R : Type u} [Ring R] [Nontrivial R] (x : R) :
 #guard_msgs(drop info) in
 example {S : Type v} [Ring S] (c : S) :
     Polynomial.nextCoeff (Polynomial.X - Polynomial.C c) = -c := by
-  rw_search [-Polynomial.nextCoeff_X_sub_C]
```

Comment:
Why did you make this change?

Reply:
The test was failing, I will revert this change and wait for the CI to see if it still fails now.

Reply:
The test still fails without this change, [507b3a5](https://github.com/leanprover-community/mathlib4/pull/16032/commits/507b3a5cad74511cfeb084e9b1dab0145654a75c).

Reply:
It looks like the test is non-deterministic: it can decide to `rw [sub_eq_neg_add]` instead of `rw [sub_eq_add_neg]`, then fail.